src/parser/ast/mod.rs

```diff
@@ -17,11 +17,25 @@ pub struct DebugSymRef {
     pub end: usize,
     /// Source file reference
     file: Arc<SimpleFile<String, String>>,
+    /// Created by the compiler
+    pub virt: bool,
```

Comment:
This could be `is_virtual`

Reply:
the problem is that virtual is a reserved word and is_virtual is the name of a getter